components/dashboards-web-component/src/gadgets-generation-wizard/components/DataPublishingComponent.jsx

```diff
@@ -139,18 +161,163 @@ class DataPublishingComponent extends Component {
+                <SelectField style={{width: '30%'}} key={selectedKey}
+                             value={selectedValue}
+                             onChange={this.handleDropDown}>
+                    {outputAttributes.map((outputAttribute, key) => {
+                        return <MenuItem key={key} value={outputAttribute} primaryText={outputAttribute}/>
+                    })}
+                </SelectField>
+                <h3 style={h3Style}>As</h3>
+                <TextField
+                    style={publishedAsValueStyle}
+                    errorText={errorTextField}
+                    onChange={this.handlePublishedAs}
+                />
+                <FloatingActionButton
+                    mini={true}
+                    style={addBtnStyle}
+                    onClick={this.addWidgetInput}
+                >
+                    <ContentAdd/>
+                </FloatingActionButton>
+                {widgetOutputConfigs.length !== 0 ?
+                    <Paper style={{
+                        marginTop: 10, marginBottom: 10
+                    }}>
+                        <Table onRowSelection={this.handleRowSelection}>
+                            <TableHeader displaySelectAll={false} adjustForCheckbox={false}>
+                                <TableRow>
+                                    <TableHeaderColumn>Publishing Value</TableHeaderColumn>
+                                    <TableHeaderColumn>Published As</TableHeaderColumn>
+                                </TableRow>
+                            </TableHeader>
+                            <TableBody displayRowCheckbox={false} deselectOnClickaway={false}>
+                                {
+                                    widgetOutputConfigs.map(function (outputAttribute, index) {
+                                        return (
+                                            <TableRow
+                                                selected={selectedRow[0] === index}>
+                                                <TableRowColumn>
+                                                    {outputAttribute.publishingValue}
+                                                </TableRowColumn>
+                                                <TableRowColumn>
+                                                    {outputAttribute.publishedAsValue}
+                                                </TableRowColumn>
+                                            </TableRow>
+                                        );
+                                    })
+                                }
+                            </TableBody>
+                        </Table>
+                        {selectedRow.length !== 0 ?
+                            <FlatButton label={<FormattedMessage id="delete.selected.row"
+                                                                 defaultMessage="Delete Selected Row"/>}
```

Comment:
This is the implementation to remove added data publishing configurations. When a row in data publish config is selected it will display a delete button for that row. This was already implemented for other charts data publishing configuration. The implementation of this was not changed and it is not used by the search bar.

Reply:
My concern is not with localizing this particular button but not localizing strings else where

Reply:
Fixed